core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -76,6 +76,9 @@ class KryoSerializerSuite extends SparkFunSuite with SharedSparkContext {
   }
 
   test("basic types") {
+    val conf = new SparkConf(false)
```

Comment:
Unit test have been updated, nothing new was found.

Tuples are registered starting from line 149 in KryoSerializer.scala.